scripts/release.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+isVersionSuccess=false
+
+echo "Start release"
+if [[ $1 == 'patch' ]]; then
+    npm version patch -m "feat: update to %s"
+    isVersionSuccess=true
+elif [[ $1 == 'minor' ]]; then
+    npm version minor -m "feat: update to %s"
+    isVersionSuccess=true
+elif [[ $1 == 'major' ]]; then
+    npm version major -m "feat: update to %s"
+    isVersionSuccess=true
+fi
+
+if [[ $isVersionSuccess == true ]]; then
+    echo "-> Release successes, start generate CHANGELOG"
+    commitTitle=$(git log --pretty=format:"%s" HEAD -1)
+    tag=$(git describe --tags "$(git rev-list --tags --max-count=1)")
+    commitBody="for more details in [CHANGELOG](https://github.com/Nek0-hinata/projectS/blob/main/CHANGELOG.md)"
+    git reset --soft HEAD^
+    pnpm changelog
+    git add .
+    git commit -m "$commitTitle" -m "$commitBody"
+
+    git tag -d "$tag"
```

Comment:
Recommend to replace [short option] with [long option] in scripts, which will more readable. When considering the user of the [long option], do not overlook applying them to other relevant commands as well.

```suggestion
    git tag --delete "$tag"
```


[long option]: https://github.com/util-linux/util-linux/blob/fa531311ab4d762afcd76f686350eeaacbe1a555/misc-utils/getopt.1.adoc#L74
[short option]: https://github.com/util-linux/util-linux/blob/fa531311ab4d762afcd76f686350eeaacbe1a555/misc-utils/getopt.1.adoc#L70